### PR TITLE
Add api

### PR DESCRIPTION
**Pull request by jadeydi**

### CODE REVIEW

File: api/_config.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# ----
+# Site
+
+title: Mixin
+description: Jekyll template for API documentation.
+url: https://developer.mixin.zone
```
Comment (cedricfung) [inline api/_config.yml]: There are several developer.mixin.zone in this request.

What we expect is developers.mixin.one/api

File: api/_config.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+collections:
+  documentation:
+    title: Documentation
+    position: 1
+  api:
+    title: APIs
+    position: 2
+  app_api:
+    title: APP APIs
+    position: 3
```
Comment (cedricfung) [inline api/_config.yml]: Are all these collections used?

File: api/_config.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  bash: Bash
+  c: C
+  css: CSS
+  html: HTML
+  java: Java
+  javascript: JavaScript
+  json: JSON
+  php: PHP
+  python: Python
+  ruby: Ruby
```
Comment (cedricfung) [inline api/_config.yml]: What are these languages? 

And should clean the config to remove useless contents.

**Comment by cedricfung** [comment]
There are also several useless files committed.

File: api/_config.yml (hunk 2 above)
Comment (jadeydi) [reply api/_config.yml]: Yeah, it is using to categorize api.

**Pull request merged**